package.json

```diff
@@ -31,19 +31,18 @@
   "dependencies": {
     "@tailwindcss/aspect-ratio": "^0.4.2",
     "@tailwindcss/forms": "^0.5.3",
-    "@tailwindcss/line-clamp": "^0.4.4",
```

Comment:
I believe I approved Mike's PR to remove this plugin since it's included in core TW since TW v3.3.3
https://github.com/tailwindlabs/tailwindcss-line-clamp/
`As of Tailwind CSS v3.3 the line-clamp utilities are now included in the framework by default and this plugin is no longer required.`


Somehow this sneaked back in probably when I merged some changes previously 😬